Tests/XcodeGenKitTests/SpecLoadingTests.swift

```diff
@@ -229,9 +229,17 @@ func specLoadingTests() {
             let options = ProjectSpec.Options(carthageBuildPath: "../Carthage/Build",
                                               createIntermediateGroups: true,
                                               bundleIdPrefix: "com.test",
-                                              developmentLanguage: "ja")
+                                              developmentLanguage: "ja",
+                                              platformVersions: PlatformVersions(iOS: "11.1", tvOS: "10.0", watchOS: "3.0", macOS: "10.12"))
             let expected = ProjectSpec(basePath: "", name: "test", options: options)
-            let parsedSpec = try getProjectSpec(["options": ["carthageBuildPath": "../Carthage/Build", "bundleIdPrefix": "com.test", "createIntermediateGroups": true, "developmentLanguage": "ja"]])
+            let dictionary: [String: Any] = ["options": [
+                "carthageBuildPath": "../Carthage/Build",
+                "bundleIdPrefix": "com.test",
+                "createIntermediateGroups": true,
+                "developmentLanguage": "ja",
+                "platformVersions": ["iOS": 11.1, "tvOS": 10.0, "watchOS": "3.0", "macOS": "10.12" ]
```

Comment:
Might be good for the test to have the case where we have 3 components to the version ("10.12.1" for example)